src/indent.cpp

```diff
@@ -1312,11 +1338,26 @@ void indent_text(void)
       {
          prev = chunk_get_prev_ncnl(pc);
          if ((prev != NULL) &&
-             (prev->type == CT_BRACE_CLOSE) &&
+             ((prev->type == CT_BRACE_CLOSE) ||
+             (cpd.settings[UO_mod_case_brace].a & AV_REMOVE)) &&
```

Comment:
maybe it's late but I don't really get this. so we do the indent only if `mod_case_brace` is true ? and why are you changing these along with adding new options ?